.build-linux/publishViewsInSchema.ps1

```diff
@@ -1,7 +1,7 @@
 Param(
```

Comment:
Does this file get copied into the linux and win directories when its built?  What I'm getting at is whether this is needed in all three places.

Reply:
It's a _little_ easier if it is, but it wouldn't be impossible to not duplicate it. It's running the .net app and doing so by looking in the folder it itself is in. If we pulled it out a level it would have to determine the OS and point to that app when running it. We could do that, I just didn't think about it tbh 🤷‍♂️. Right now it's set up in the .net app's csproj as a file to always publish/build with, so it was just kind of automatic and I didn't give it a second glance.

Reply:
I'm totally fine with that.  I didn't know if it was copied during build or if it was manually put in all three spots.  It sounds like it can stay the way it is though.